pre_commit_hooks/mixed_line_ending.py

```diff
@@ -0,0 +1,250 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+from enum import Enum
```

Comment:
in python 2 we'll need to declare this as a dependency.

The magic to doing this is to add this to setup.py:

```python
extras_require={':python_version=="2.7"': ['enum34']}
```

I actually [made a video on this topic](https://www.youtube.com/watch?v=rS2ZgWedTUs) as well :D